docs/i18n/portuguese/how-to-translate-the-website.md

```diff
@@ -21,7 +20,7 @@ The `meta-tags.json` file contains the information for our website's meta tag in
 
 To add a new language, create a folder with the language name as the title next to the other languages and copy the JSON files from another language into your new folder.
 
-In the `all-langs.js` file, add the language to the `client` array in the first variable. Then, follow the instructions in the comments to add the rest of the necessary variables.
+In the `allLangs.js` file, add the language to the `client` array in the first variable. Then, follow the instructions in the comments to add the rest of the necessary variables.
```

Comment:
We should push up the English changes first, so this correctly downloads as `all-langs.js`.

Reply:
This is `allLangs.js` in the current English source and is correct